AndelaOSP/art-dashboard
# #160192048 Allow only admin users to access the site

## What does this PR do?

Restricts access to the app to users that have admin privileges only

## Description of Task to be completed?

```gherkin
Scenario: Only admins can access the site.
Given a user with no admin rights
When I login to the ART site
Then I  should see an error message 
And be redirected to the login page.
```

## How should this be manually tested?

* Login to the application
* If you are not an admin user, you should see a modal telling you that you are not authorized to access the site

## What are the relevant pivotal tracker stories?

[160192048](https://www.pivotaltracker.com/story/show/160192048)

## Any background context you want to add?

Admin privileges are defined on Firebase. Make sure that your user has the appropriate custom claims.

## Important notes

N/A

## Packages installed

N/A

## Deployment note

N/A

## Related PRs branch | PR (branch_name) | (pr_link)

Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos


## Screenshots

<img width="1586" alt="screenshot 2018-11-16 at 10 10 34" src="https://user-images.githubusercontent.com/31339738/48611303-0cdddc00-e997-11e8-8217-35e0f2cde310.png">
